reference/5.1/Microsoft.PowerShell.Utility/Out-File.md

```diff
@@ -356,15 +359,14 @@ You can pipe any object to `Out-File`.
 
 ## NOTES
 
-The `Out` cmdlets do not format objects; they just render them and send them to the specified
-display destination. If you send an unformatted object to an `Out` cmdlet, the cmdlet sends it to a
-formatting cmdlet before rendering it.
+If you send an unformatted object to an `Out` cmdlet, the cmdlet sends it to a formatting cmdlet
+before rendering it.
```

Comment:
I fear that this may still cause confusion ("rendering" is not typically applied to file operations). 

Given your addition to the "DESCRIPTION" section above re same representation as in the terminal, which clarifies the fundamental behavior, I suggest framing the information differently, along the lines of:

Input objects are _automatically_ formatted on saving, as they would be in the terminal, but you may use a `Format-*` cmdlet explicitly to control the formatting "shape" (e.g., `Get-Date | Format-List | Out-File out.txt`)

Reply:
Let me know what you think of my changes here on the latest commit